capstone/src/main/java/com/google/univiz/scorecard/ScorecardData.java

```diff
@@ -8,10 +8,12 @@
  * JSON data directly from the CollegeScorecard API.
  */
 @AutoValue
```

Comment:
To get this to work with gson you'll also need to annotate this class with `@GenerateTypeAdapter`.